A1/A1.ino

```diff
@@ -0,0 +1,67 @@
+#include <Smartcar.h>
+
+//const int SIDE_FRONT_PIN = ; // you can use only analog enabled pins
+const int TRIGGER_PIN    = 6; // D6
+const int ECHO_PIN       = 7; // D7
+
+ArduinoRuntime arduinoRuntime;
+SR04 front(arduinoRuntime, TRIGGER_PIN, ECHO_PIN, 500);
+//GP2D120 sideFrontIR(arduinoRuntime,SIDE_FRONT_PIN); // measure distances between 5 and 25 centimeters
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+const int fSpeed   = 70;  // 70% of the full speed forward
+const int bSpeed   = -70; // 70% of the full speed backward
+const int lDegrees = -75; // degrees to turn left
+const int rDegrees = 75;  // degrees to turn right
+
+SimpleCar car(control);
+void setup()
+{
+    Serial.begin(9600); // start the serial
+}
+
+void loop()
+{
+    Serial.println(front.getDistance());
+    if (front.getDistance() < 100)
+    {
+      if(front.getDistance() > 0){
+          car.setSpeed(0);
+       }
```

Comment:
What happens if the first call to getDistance returns 0 and the second one returns 200? The car will incorrectly stop.
Also if you are changing the name of the sketch, you should update that name change on the workflow so your code keeps getting built on CI.